.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SHELLCHECK_OPTS: -e SC1004 # exclude some shellcheck warnings.
-          SHFMT_OPTS: -s # arguments to shfmt.
+          SHFMT_OPTS: -s -i 2 -bn -sr -ln bash # shfmt: simplify (s), 2-space indentation (-i 2), braces on same line (-bn), space after redirects (-sr), bash dialect (-ln bash)
```

Comment:
As I recall, you're preference was indenting 4 spaces, mine was 3 spaces and shellcheck's default was 12.  So, how did we end up with 2? ;-)  I think 2 is too little and 12 is way too much.

Reply:
changed to 4